tests/integration/test_logcollector/test_log_format/test_log_format_values.py

```diff
@@ -0,0 +1,472 @@
+
+    Raises:
+        TimeoutError: If the "Analyzing file" callback is not generated.
+    """
+    wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+    if cfg['log_format'] == 'eventchannel' or cfg['log_format'] == 'eventlog':
+        log_callback = logcollector.callback_eventchannel_analyzing(cfg['location'])
+    else:
+        log_callback = logcollector.callback_analyzing_file(cfg['location'])
+    wazuh_log_monitor.start(timeout=5, callback=log_callback,
+                            error_message=logcollector.GENERIC_CALLBACK_ERROR_ANALYZING_FILE)
+
+
+def check_log_format_value_valid(conf):
+    """
+    Check if Wazuh runs correctly with the correct log format and a specific content.
+
+    Raises:
+        TimeoutError: If the "Analyzing file" callback is not generated.
+    """
+
+    wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+    if conf['log_format'] not in log_format_windows_print_analyzing_info:
+        if conf['log_format'] in log_format_not_print_reading_info:
+            # Logs format that only shows when a specific file is read.
+
+            log_callback = logcollector.callback_read_file(conf['location'])
+            wazuh_log_monitor.start(timeout=5, callback=log_callback,
+                                    error_message=logcollector.GENERIC_CALLBACK_ERROR_READING_FILE)
+
+        elif conf['log_format'] == 'multi-line:3':
+            # It is necessary to reorganize the content of the file to compare it
+            # with the generated output of the 'multiline' format.
+
+            msg = ""
+            with open(location, 'r') as file:
+                for line in file:
+                    msg += line.rstrip('\n')
+                    msg += ' '
+                log_callback = logcollector.callback_reading_file(conf['log_format'], msg.rstrip(' '))
+                wazuh_log_monitor.start(timeout=5, callback=log_callback,
+                                        error_message=logcollector.GENERIC_CALLBACK_ERROR_READING_FILE)
+        else:
+            # Verify that the content of the parsed file is equal to the output generated in the logs.
+            with open(location, 'r') as f:
+                lines = f.readlines()
+                for line in lines:
+                    log_callback = logcollector.callback_reading_file(conf['log_format'], line.rstrip('\n'))
+                    wazuh_log_monitor.start(timeout=5, callback=log_callback,
+                                            error_message=logcollector.GENERIC_CALLBACK_ERROR_READING_FILE)
+
+    elif conf['log_format'] == 'iis':
+        log_callback = logcollector.callback_read_file(conf['location'])
+        wazuh_log_monitor.start(timeout=5, callback=log_callback,
+                                error_message=logcollector.GENERIC_CALLBACK_ERROR_READING_FILE)
+
+
+def analyzing_invalid_value(conf):
+    """
+    Checks for the error message that is shown when the record format type is valid but the content is invalid.
+    """
```

Comment:
Follow the standard for the docstings. Also, the `Args` and `Returns` fields are missing.